core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -1495,6 +1510,11 @@ public SELF withCreateContainerCmdModifier(Consumer<CreateContainerCmd> modifier
         return self();
     }
 
+    public SELF withCreateContainerCmdCustomizer(CreateContainerCmdCustomizer customizer) {
```

Comment:
We should think about either deprecating `withCreateContainerCmdModifier` or adding an overload, to avoid increasing the API surface. Maybe, as a first step, we could keep `withCreateContainerCmdModifier` only but make it populate `localCreateContainerCustomizers`?

Also, if we name it `CreateContainerCmdModifier`, then we could align both APIs and not have to deprecate the old one (just thinking out loud...) - WDYT?

Reply:
looks like I forgot something when trying to overload `withCreateContainerCmdModifier` 🙈. Now, this is done. My only question is about the location where `createContainerCmdCustomizers` is being called because `createContainerCmdModifiers` is currently called before. I think the current implementation is ok and we should drop `createContainerCmdModifiers` references due to are not being used.